python-sdk/tests/files/type/test_base.py

```diff
@@ -1,13 +1,17 @@
+from unittest.mock import patch
+
 from astro.constants import FileType
 from astro.files import File
 
 
-def test_str():
+@patch("astro.files.locations.base.BaseFileLocation.validate_conn", return_value=None)
+def test_str(validate_conn):
     file = File(path="astro", conn_id="local", filetype=FileType.CSV)
     assert file.__str__() == "astro"
 
 
-def test_repr():
+@patch("astro.files.locations.base.BaseFileLocation.validate_conn", return_value=None)
```

Comment:
shouldn't be needed

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1478/commits/b89f84be043188126f642e50de139bf330adb72b)